src/ui/components/TestSuite/views/TestRecording/TestRecordingEvents/UserActionEventRow.tsx

```diff
@@ -55,8 +56,9 @@ export default memo(function UserActionEventRow({
   userActionEvent: UserActionEvent;
 }) {
   const { data } = userActionEvent;
-  const { command, error, parentId, timeStampedPoints, resultVariable } = data;
-  const { result: resultTimeStampedPoint } = timeStampedPoints;
+  const testRunnerName = groupedTestCases.environment.testRunner.name;
+  const { command, error, parentId } = data;
+  const resultPoint = getUserActionTestEventResultPoint(userActionEvent, testRunnerName);
```

Comment:
Is this something we could smooth over in the `RecordingTestMetada` component? so we didn't have to make lower level views aware of the implementation detail?

Reply:
Possibly. It's a mixture of two different checks: does it have a possible execution point that we want to use, _and_ is it a relevant test step.  I could add another field to the test event that saves that value, but not sure if it's worth doing that.